platform/broadcom/sonic-platform-modules-ruijie/common/modules/i2c-mux-pca9641.c

```diff
@@ -0,0 +1,645 @@
+/*
+ * I2C multiplexer driver for PCA9541 bus master selector
```

Comment:
can you use the one in linux kernel?

Reply:
Will revise I2C deadlock problem when use 9548 and 9541 at the same time.

Reply:
same with above, Will revise I2C deadlock problem when use 9548 and 9541 at the same time.